oxenstored/select_stubs.c

```diff
@@ -66,19 +66,16 @@ stub_select_on_poll (value fd_events, value timeo)
 
                             if (c_fds[i].revents & POLLNVAL)
                                     unix_error (EBADF, "select", Nothing);
-                            Field (events, 0) =
+                            Field (events, 2) =
```

Comment:
On lines 47 and 48 POLLPRI is written to, sholud the remove field be reinstated, or is it better to remove the write to it? otherwise nasty surprises may happen when the code is change